src/ui/utils/mixpanel.ts

```diff
@@ -6,13 +6,89 @@ import { isReplayBrowser, skipTelemetry } from "./environment";
+  | ["breakpoint.preview_has_hits"]
+  | ["breakpoint.preview_no_hits"]
+  | ["breakpoint.remove"]
+  | ["breakpoint.set_condition"]
+  | ["breakpoint.set_log"]
+  | ["comments.create"]
+  | ["comments.delete"]
+  | ["comments.focus"]
+  | ["comments.select_location"]
+  | ["comments.start_edit"]
+  | ["console.clear_messages"]
+  | ["events_timeline.select"]
+  | ["events_timeline.select_source"]
+  | ["error.unauthenticated_viewer"]
+  | ["error.unauthorized_viewer"]
+  | ["header.open_share"]
+  | ["header.edit_title"]
+  | ["gutter.add_comment"]
+  | ["key_shortcut.full_text_search"]
+  | ["key_shortcut.show_command_palette"]
+  | ["key_shortcut.toggle_left_sidebar"]
+  | ["layout.default_devtools"]
+  | ["layout.default_viewer"]
+  | ["layout.devtools"]
+  | ["layout.viewer"]
+  | ["launch.download_replay", { OS: "mac" | "linux" | "windows" }]
+  | ["login.first_log_in"]
   | ["net_monitor.add_type", { type: CanonicalRequestType }]
   | ["net_monitor.delete_type", { type: CanonicalRequestType }]
   | ["net_monitor.open_network_monitor"]
   | ["net_monitor.seek_to_request"]
-  | ["net_monitor.select_request_row"];
+  | ["net_monitor.select_request_row"]
+  | ["object_inspector.label_click"]
+  | ["onboarding.created_team"]
+  | ["onboarding.demo_replay_launch"]
+  | ["onboarding.demo_replay_prompt"]
+  | ["onboarding.download_replay", { OS: "mac" | "linux" | "windows" }]
+  | ["onboarding.download_replay_prompt"]
+  | ["onboarding.finished_onboarding"]
+  | ["onboarding.invited_team_member"]
+  | ["onboarding.launch_replay"]
+  | ["onboarding.skipped_create_team", { skippedFrom: string }]
+  | ["onboarding.skipped_replay_download"]
+  | ["onboarding.started_onboarding"]
+  | ["onboarding.team_invite"]
+  | ["quick_open.open_quick_open"]
+  | ["session.devtools_start", { userIsAuthor: boolean }]
+  | ["session_end", { reason: string }]
+  | ["session_start", { workspaceId: WorkspaceId | null }]
+  | ["share_modal.copy_link"]
+  | ["share_modal.set_private"]
+  | ["share_modal.set_public"]
+  | ["share_modal.set_team"]
+  | ["share_modal.set_team"]
+  | ["share_modal.set_team"]
+  | ["share_modal.set_team"]
+  | ["share_modal.set_team"]
+  | ["share_modal.set_team"]
+  | ["share_modal.set_team"]
+  | ["share_modal.set_team"]
+  | ["team_change", { workspaceId: WorkspaceId | null }]
+  | ["timeline.comment_select"]
+  | ["timeline.marker_select"]
+  | ["timeline.pause"]
+  | ["timeline.play"]
+  | ["timeline.progress_select"]
+  | ["timeline.replay"]
+  | ["toolbox.secondary.editor_toggle"]
+  | ["toolbox.secondary.video_toggle"]
+  | ["toolbox.toggle_sidebar"]
+  | ["upload.complete", { sessionId: SessionId }]
+  | ["upload.create_replay", { isDemo: boolean }]
+  | ["upload.discard", { isDemo: boolean }]
+  | ["user_options.launch_replay"]
+  | ["user_options.select_docs"]
+  | ["user_options.select_settings"]
+  | [`toolbox.primary.${PrimaryPanelName}_select`]
+  | [`toolbox.secondary.${SecondaryPanelName}_select`];
```

Comment:
Can move all `toolbox.` up to be together.

Reply:
yeah `sort` got me because of the different quotes